app/metrics.yaml

```diff
@@ -23,3 +23,152 @@ distribution:
     notification_emails:
       - dmu@mozilla.com
     expires: "2020-05-01"
+pages:
+  page_load:
+    type: timing_distribution
+    lifetime: application
+    description: >
+      Counting how long each page takes to load.
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2230
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+    notification_emails:
+      - dmu@mozilla.com
+    expires: "2020-05-01"
+url:
+  domains:
+    type: counter
+    lifetime: application
+    description: >
+      Counting how many domains are visited in a session.
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2230
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+    notification_emails:
+      - dmu@mozilla.com
+    expires: "2020-05-01"
+  visits:
+    type: counter
+    lifetime: application
+    description: >
+      Counting how many URL links are visited in a session.
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2230
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+    notification_emails:
+      - dmu@mozilla.com
+    expires: "2020-05-01"
+  insert_types:
+    type: labeled_counter
+    lifetime: application
+    description: >
+      Counting how many domains and URL are visited in a session.
+    labels:
+      - type_link
+      - type_query
+      - voice_query
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2230
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+    notification_emails:
+      - dmu@mozilla.com
+    expires: "2020-05-01"
+durarion:
+  immersive_mode:
+    type: timing_distribution
+    lifetime: application
+    description: >
+      Counting how long users stay at the immersive mode.
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2230
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+    notification_emails:
+      - dmu@mozilla.com
+    expires: "2020-05-01"
+  window_lifetime:
```

Comment:
What about a `window` category with a `opened_duration` or something like that?

Reply:
I am going to move this to another PR because of blockers.